src/plugins/graph/models/graph-model.ts

```diff
@@ -92,17 +92,17 @@ export const GraphModel = TileContentModel
     annotationSizesCache: new ObservableMap<string,RectSize>()
   }))
   .preProcessSnapshot((snapshot: any) => {
+    const newSnap = cloneDeep(snapshot);
```

Comment:
There's a slight inelegance/inefficiency here in that you're always cloning even though you don't always need to, but as to whether that's worth addressing: 🤷 .